generator/src/main/java/org/corfudb/generator/Generator.java

```diff
@@ -0,0 +1,75 @@
+package org.corfudb.generator;
+
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+import org.corfudb.generator.operations.CheckpointOperation;
+import org.corfudb.generator.operations.Operation;
+import org.corfudb.generator.operations.RemoveOperation;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.exceptions.NetworkException;
+import org.corfudb.runtime.exceptions.TransactionAbortedException;
+import org.corfudb.runtime.view.stream.IStreamView;
+
+import static java.util.concurrent.Executors.newWorkStealingPool;
+
+/**
+ * Created by maithem on 7/14/17.
+ */
+public class Generator {
+    public static void main(String[] args) {
+        String endPoint;
+        int numStreams;
+        int numKeys;
+        long cpPeriod;
+        int numThreads = 8;
+        int numOperations;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "numOperations" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)